extensions/ql-vscode/src/remote-queries/download-link.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Represents a link to an item to be downloaded. 
```

Comment:
Are these all GitHub actions _artifacts_? If so, are they always archive files that need to be unzipped before using them?
```suggestion
 * Represents a link to an artifact to be downloaded. 
```

Reply:
Also below, can we standardize on using `artifact` instead of file or item?

It's not a major point. I'm just trying to clarify terms.

Reply:
It's a good point, and agree terms should be clear! I didn't want to make it too specific to Actions but I think I just made it confusing. I've updated to use artifact!